src/components/MusicMd/Render.tsx

```diff
@@ -29,6 +30,7 @@ interface MarkdownItWithMMD extends MarkdownIt {
   setMaxWidth: (maxWidth: number) => void;
   meta: {
     youTubeId: string;
+    autoScroll: string;
```

Comment:
TL;DR: Please refactor the implementation of auto-scroll to work and behave the way zoom works.

While I like the idea of associating the auto-scroll as a property of the song, I'd instead like for it to be associated with the device. 

We can implement the auto-scroll similar to how transpose or zoom works. Add some widgets to the "more" menu to configure the autoscroll. Once you've configured your desired scroll speed for a given song, it will be saved in local storage of the device. And the next time that the song is loaded, the previously associated scroll speed will be loaded.

There are slight behavior differences between transpose and zoom. Transpose is stored as an argument in the URL. This way, if you share the song via a QR code, the current transpose setting is also shared. I believe auto-scroll should work the way zoom works, because different people have different devices and zoom levels, and will want different auto-scroll settings on a per-device basis.

So, if you have the time, I'd love to see this PR refactored to implement auto-scroll in a way that is consistent with how zoom is implemented.

Again, thanks so much for the contribution.